index.html

```diff
@@ -1,3 +1,134 @@
+                    <li>2015</li>
+                </ul>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <ul class="grid1">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button>See Project</button>
+
+            </div>
+        </div>
+
+        <div class="main-section">
+            <div class="main-image">
+                <img src="img/markus-spiske-ZSZ6wzNU12Q-unsplash.jpg" alt="main image">
+            </div>
+            <div class="text-section">
+                <h1>Tonic</h1>
+                <ul class="navi">
+                    <li>Canopy</li>
+                    <li>Black end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <ul class="grid1">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button>See Project</button><br>
+
+            </div>
+            <br>
+        </div>
+
+        <div class="main-section">
+            <div class="main-image">
+                <img src="img/markus-spiske-ZSZ6wzNU12Q-unsplash.jpg" alt="main image">
+            </div>
+            <div class="text-section">
+                <h1>Tonic</h1>
+                <ul class="navi">
+                    <li>Canopy</li>
+                    <li>Black end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <ul class="grid1">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button>See Project</button>
+
+            </div>
+        </div>
+        <div class="about-section">
```

Comment:
- [ ] I think it's better to use the `<section>` tag for the about section instead of the div tag. To make your project more semantically correct.